### PR TITLE
jsk_interactive_marker: use catkin_install_python in CMakeLists.txt

### PR DESCRIPTION
**Pull request by k-okada**

### CODE REVIEW

File: jsk_interactive_markers/jsk_interactive_marker/setup.py
Comment (mqcmd196) [inline jsk_interactive_markers/jsk_interactive_marker/setup.py:1]: Why do we need `setup.py` and `catkin_python_setup()` ? Only for `joint_state_publisher.py` ??

Comment (pazeshun) [reply jsk_interactive_markers/jsk_interactive_marker/setup.py:1]: cf. https://github.com/jsk-ros-pkg/jsk_recognition/pull/2851

Comment (k-okada) [reply jsk_interactive_markers/jsk_interactive_marker/setup.py:1]: oh, I didn't know `catkin_install_python()` runs without `catkin_python_setup()`. thank you

Comment (k-okada) [reply jsk_interactive_markers/jsk_interactive_marker/setup.py:1]: moved to https://github.com/jsk-ros-pkg/jsk_visualization/pull/897

**Pull request closed without merging**